routes/customer.js

```diff
@@ -8,7 +8,7 @@ const use = (fn) => (req, res, next) =>
 
 //Admin
 router.post("/adminLogin", use(adminCustomersController.adminLogin));
-router.post("/signup", use(adminCustomersController.signup));
+router.post("/signup", use(adminCustomersController.signUp));
```

Comment:
give correct spell for signup in both place

Reply:
resolved